src/wavm/io.cpp

```diff
@@ -665,6 +666,15 @@ namespace wasm {
     // Unsupported
     // -----------------------------
 
+    WAVM_DEFINE_INTRINSIC_FUNCTION(wasi, "fd_renumber", I32, wasi_fd_renumber, 
+            I32 fdOld, I32 fdNew) {
+        throwException(Runtime::ExceptionTypes::calledUnimplementedIntrinsic);
+    }
+
+    WAVM_DEFINE_INTRINSIC_FUNCTION(env, "tmpfile", I32, tmpfile) {
+        throwException(Runtime::ExceptionTypes::calledUnimplementedIntrinsic);
+    }
+
```

Comment:
@csegarragonz we often have to add functions like this to keep the linker happy but they're never actually called at runtime (and they'll throw an exception if they are, at which point we can think about implementing them).